src/components/NotFoundPage.js

```diff
@@ -3,13 +3,22 @@ import Header from './Header';
 import Body from './Body';
 import Footer from './Footer';
 import NotFound from './NotFound';
+import { FormattedMessage } from 'react-intl';
 
-const NotFoundPage = () => {
+const NotFoundPage = ({ slug }) => {
   return (
     <div className="NotFoundPage">
+      <style jsx>{`
+        p {
+          margin: 4rem 0;
+        }
+      `}</style>
       <Header />
       <Body>
         <NotFound />
+        { slug &&
+        <p align="center"><FormattedMessage id="notFound.search" defaultMessage="Let me try to find {term} for you..." values={{ term: slug }} /></p>
```

Comment:
Just FYI, `align` is not HTML5 valid. It will work for now, but you know.

Reply:
ah good to know. I'll use `<center></center>`